src/drove.go

```diff
@@ -485,6 +485,28 @@ func writeConf() error {
 func nginxPlus() error {
 	config.RLock()
 	defer config.RUnlock()
+
+	logger.WithFields(logrus.Fields{
+		"nginx": config.Nginxplusapiaddr,
+	}).Info("endpoint")
+
+	endpoint := "http://" + config.Nginxplusapiaddr + "/api"
+	//Create transport here for connection re-use
+	tr := &http.Transport{
+		MaxIdleConns:       30,
+		DisableCompression: true,
+	}
+
+	client := &http.Client{Transport: tr}
```

Comment:
We should create and reuse the transport client etc only once

Reply:
Will include it in a serparate rewrite built upon tushar's patches along with HAProxy support